CorsixTH/Lua/audio.lua

```diff
@@ -374,11 +374,25 @@ function Audio:stopSound(sound)
     return
   end
 
-  if sound.channel and sound.channel >= 0 then
-    sound_fx:stop(sound.channel, sound.played_callback_id)
+  if sound and sound.handle then
+    sound_fx:stop(sound.handle, sound.played_callback_id)
   end
 end
 
+--! Determine if a given sound is still playing
+--!param sound (table) The `sound` table returned by `Audio:playSound`.
+function Audio:isPlaying(sound)
```

Comment:
Could this also have a pass through function in UI like the others?

Reply:
No longer used, so not for now.